Unix/cloc

```diff
@@ -6519,7 +6520,7 @@ sub blank_regex {                            # {{{1
     my ($language) = @_;
 
     print "-> blank_regex(language=$language)\n" if $opt_v > 2;
-    
+
```

Comment:
Vscode removing trailing whitespace BTW.  Let me know if this is unwanted change though.